README.md

```diff
@@ -179,6 +179,38 @@ make timed_run
 make stop_slaves  # mac: killall python
 ```
 
+#### Profiling
+
+In case you need to profile different the application to know which part of the code are more time consuming, there are two different profilers available. Both will store the profiling output in a directory that's determined by the `PROFILE_DIR` env variable. If  this variable is not set, the output files will be store in a folder called profiler inside the OS temp folder (`/tmp/profiler` usually)
```

Comment:
`/tmp/profiler` -> `/tmp/profile`

If `/tmp` is typically owned by `root` then it may be worth mentioning later that snakeviz will need you to chown `/tmp/profile` to your user.

Reply:
It also may be helpful to note to the user that they may access the HTML files with the `file:///tmp/path` and do not need snakeviz in that case.